main/test/ca/uwaterloo/flix/library/TestList.flix

```diff
@@ -2915,4 +2915,18 @@ def literal05(): Bool =
 def literal06(): Bool =
     not List.isEmpty(List#{{}})
 
+/////////////////////////////////////////////////////////////////////////////
+// frequency                                                               //
+/////////////////////////////////////////////////////////////////////////////
+@Test
+def frequency01(): Bool =
+    let l = List#{"a"};
+    let m = Map#{"a" => 1};
+    Assert.eq(m, List.frequency(l))
+
+@Test
+def frequency02(): Bool =
+    let l = List#{"b", "a", "c", "a", "b", "a"};
+    let m = Map#{"a" => 3, "b" => 2, "c" => 1};
+    Assert.eq(m, List.frequency(l))
```

Comment:
I think we usually have 5-8 tests, and at least one of them should use the empty map